### ISSUE TITLE
Car Following Logic Incorrect

### ISSUE THREAD

**Issue by tlandle-intel** [ucla-mobility/OpenCDA#227]
In the car following logic, the following is true:

Full brake occurs in following scenario:

- If car detected to be following another car AND the distance to closest object is less than 3 meters, do not move
- Car following flag set true when:
    - Any object is detected in front of the vehicle within radius circle of ~5 m from center of car. 
          - This includes cars in the lane next to it or behind
    - AND 
    - Overtake is not allowed
    - Overtake is not allowed when
         - Car is detected near an intersection
- To summarize:
        - Cars will brake and stay braked if a car is in the lane next to them while they are approaching an intersection. This is faulty behavior for obvious reasons. 

Is there any WA for this behavior?
